config/jobs/kubernetes/sig-network/dualstack-e2e.yaml

```diff
@@ -141,7 +141,7 @@ periodics:
       - --aksengine-location=westus2
       - --aksengine-public-key=$(AZURE_SSH_PUBLIC_KEY_FILE)
       - --aksengine-template-url=https://raw.githubusercontent.com/Azure/aks-engine/master/examples/dualstack/kubernetes.json
-      - --aksengine-download-url=https://aka.ms/aks-engine/aks-engine-k8s-e2e.tar.gz
+      - --aksengine-download-url=https://github.com/Azure/aks-engine/releases/download/nightly/aks-engine-nightly-linux-amd64.tar.gz
```

Comment:
can we pin to a specific nightly? or is this ok?
I'm afraid we break for some unrelated change

Reply:
@aojea I share the same concern. The locked version unfortunately doesn't have the fix yet. Once https://github.com/kubernetes-sigs/cloud-provider-azure/pull/571 merges later today we can change back to using the old version. I want to unblock today's test run until the other PR is merged. Or we can just wait for the other one to merge and not do this. WDYT?

Reply:
I think that is better to wait then, so we can traces the changes, if we do several changes at the same time, and after that the job fails,  we'll not be able to reconstruct what happened

Reply:
sgtm! I'll close this PR and we'll just get the other one merged.